src/Aspire.Hosting.Seq/SeqBuilderExtensions.cs

```diff
@@ -36,11 +36,51 @@ public static IResourceBuilder<SeqResource> AddSeq(
             .WithImage(SeqContainerImageTags.Image, SeqContainerImageTags.Tag)
             .WithImageRegistry(SeqContainerImageTags.Registry)
             .WithEnvironment("ACCEPT_EULA", "Y")
+            .WithEnvironment("SEQ_FIRSTRUN_NOAUTHENTICATION", "True")
```

Comment:
Disabling auto-merge until @eerhardt or @sebastienros approve these changes.

Reply:
For context, this is in reaction to some changes in docker image 2025-2. Release notes are [here](https://datalust.co/docs/whats-new), but the specific section is:

> Require a password, or opt-out, when initializing a new instance: to protect from various failure modes related to instance/container storage, a password or explicit opt-out is required when initializing new Seq instances (see "Breaking changes" below).